tuf/ngclient/updater.py

```diff
@@ -122,12 +126,38 @@ def refresh(self) -> None:
             RepositoryError: Metadata failed to verify in some way
             DownloadError: Download of a metadata file failed in some way
         """
+        #                                     json.load()
+        # Load self.spec_version using self._load_local_metadata() here from disk
+        # Exception would be raised if spec_version is not found locally. Set to "1" in that case
+        repository_versions = self._get_repository_versions()
+
+        # Updating self.spec_version
+        specver_message = _get_spec_version(
```

Comment:
Use [tuple unpacking](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences):

```python
spec_version, message =  _get_spec_version(...)
```